modules/KalturaSupport/kalturaIframeClass.php

```diff
@@ -578,6 +578,19 @@ private function getEntryTitle(){
 		return "Kaltura Embed Player iFrame";
 	}
 
+	function outputCompatibilityTags(){
+		if (isset($this->request) && !empty($this->request)) {
```

Comment:
@vladm3 - any reason for the added security check here?
In general we have an IoC that injects the request into the kalturaIframeClass, so unless you saw it's necessary please remove the redundant check.